### PR TITLE
Back button & correct fields red on registration

### LINKED ISSUE #348: Only make fields red that need changing in Registration form.

### LINKED ISSUE #353: Registration hangs
Not sure if this is a front-end or back-end issue, but currently when registering, the page hangs when clicking the 'register' button (with correct filled in information)

### PR DESCRIPTION
**Pull request by LarsDoorenbos**
Fixes #348 
Fixes #353

### CODE REVIEW

**Comment by LarsDoorenbos** [comment]
Now also includes #353

**Comment by Qqwy** [comment]
Fixes #353

File: pydash-front/src/registration/RegistrationPage.js [hunk 1]
```diff
@@ -114,14 +127,42 @@ class RegistrationPage extends Component {
         e.preventDefault()
```
Comment (Qqwy) [inline pydash-front/src/registration/RegistrationPage.js]: This function now is ridiculously long. It does too much: 

- preventing normal form submission
- validation of required fields
- validation of email
- validation of password
- validation of password confirmation
- sending the request to the server
- checking the server's response
- showing success/error based on the server's response.

Try to split it up. :slightly_smiling_face:

Comment (Qqwy) [reply pydash-front/src/registration/RegistrationPage.js]: Oh, and the name `tryLogin` is obviously very weird for the registration page.

Comment (LarsDoorenbos) [reply pydash-front/src/registration/RegistrationPage.js]: Also done, similar to the previous one the functions are still a bit long but I feel they do their own thing and only that thing. Do you agree?

**Comment by InputUsername** [comment]
@TheVeggydude did you refresh? The old stuff might still be cached. I could see the back button & red fields after refreshing.

**Pull request merged**